src/Library/demos/Picture/main.blp

```diff
@@ -0,0 +1,51 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: _("Picture");
+  description: _("Display images at their natural sizes");
+
+  child: Box {
+    orientation: vertical;
+
+    LinkButton {
+      label: _("API Reference");
+      uri: "https://docs.gtk.org/gtk4/class.Picture.html";
+      margin-bottom : 12;
+    }
+
+    Adw.Clamp {
+      maximum-size: 500;
+
+      ListBox {
```

Comment:
selection mode none please to avoid the blue "selected" overlay:

![image](https://github.com/sonnyp/Workbench/assets/19673/1a9a0c1d-15a4-4d08-a8b4-3247009cb97c)